include/deal.II/base/vectorization.h

```diff
@@ -3841,9 +3841,25 @@ inline DEAL_II_ALWAYS_INLINE VectorizedArray<Number, width>
   return VectorizedArray<Number, width>() - u;
 }
 
+/**
+ * Output operator for vectorized array.
+ *
+ * @relatesalso VectorizedArray
+ */
+template <typename Number>
+inline std::ostream &
+operator<<(std::ostream &out, const dealii::VectorizedArray<Number> &p)
```

Comment:
When is `operator<<` and `print()` used in deal.II? 

I find the `print` methods quite useful because you can add additional parameters like formatting. Furthermore, it would be quite useful if we could specify which/how many entries we want to print. E.g. `Matrixfree::n_active_entries_per_cell_batch()` and  `Matrixfree::n_active_entries_per_face_batch()` provides us the information about how many lanes are actually relevant.

Reply:
I can implement it in a follow up PR if want this feature.

Reply:
My motivation here was to also be able to print the content where `VectorizedArray` is a nested type, e.g. `Tensor<1, dim, VectorizedArray<double>>`. In also see, `VectorizedArray` more like a `Tensor` object with a fixed and rather small number of entries (where we provide `operator<<`) than like a matrix or a vector (where we typically have a `print()` interface).